functions/users.inc

```diff
@@ -514,19 +514,18 @@ function createPassword(/*.string.*/$id, /*.string.*/$password): string
         $duration = '+1 year';
     }
 
-    $value = lookup_user_by_login($id);
+    $value = lookup_users_by_email($id);
```

Comment:
Are you sure this is what we want to do? 
Not using `lookup_users_by_login`?
I see the very first call in `lookup_users_by_login` is `lookup_users_by_email` but it also then fallsback to the CRM login field if it exists.

Reply:
Interesting, when this was `lookup_users_by_login`, the Docker console output complained that the function was undefined. Perhaps it was another part of this that had been failing, I will double check.

I do know that changing to `lookup_users_by_email` did result in successfully completing the POST to `configure_system`. I'll check into this and report back.

Reply:
I can see why it had been failing before, as well. `lookup_user_by_login` was missing the `s` in `users`, so that probably explains the failure.

Reply:
Also, I'm curious, but it seems like this function is only used by `configure_system` and not anywhere else. Is there a reason this might be the case?